### DISCUSSION TITLE
Allow reloading of assets without having to restart the game.

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#18102] [Feature requests]
It'd be very nice for people making park objects (or upcoming asset packs) if there'd be a reload button so you don't have to restart the game and load your test map, etc every single time you change a parameter. Would also make the workflow easier for people searching for objects while building a park on something like NEDesigns so they can add the new object without needing to restart their game.

**Comment by IntelOrca** [comment]
More or less this:
https://github.com/OpenRCT2/OpenRCT2/issues/11439

**Reply by karst** [reply]
Pretty much yea! I didn't find it in discussions, forgot to look in issues :P
